lib/isFQDN.js

```diff
@@ -44,6 +44,11 @@ function isFQDN(str, options) {
 
     if (/[\s\u2002-\u200B\u202F\u205F\u3000\uFEFF\uDB40\uDC20]/.test(tld)) {
       return false;
+    } // disallow special characers
+
+
+    if (/[\u00A9\uFFFD]/.test(tld)) {
```

Comment:
Is this an exhaustive list or we might be having more characters? And why don't we add them on line 45?